src/shared-components/selectorButton/test.js

```diff
@@ -175,5 +175,20 @@ describe('<SelectorButton />', () => {
       wrapper.simulate('click');
       expect(spy).toHaveBeenCalled();
     });
+
+    it('Does nothing when no onClick is set', () => {
+      const wrapper = shallow(<SelectorButton checked={false} />);
```

Comment:
This is helpful - I didn't realize that you could pass `undefined` as a React event handler without breaking stuff.

Reply:
I wasn't positive (hence the test).  But if you don't pass an argument it's undefined so I figured that would be fine here (and eslint was getting angry about an empty arrow function).

Reply:
Yeah, that makes sense. To think of all those hours I've wasted defining empty arrow function 🤦